elements/pl-multiple-choice/pl-multiple-choice.py

```diff
@@ -11,9 +11,10 @@
 def prepare(element_html, data):
     element = lxml.html.fragment_fromstring(element_html)
     required_attribs = ['answers-name']
-    optional_attribs = ['weight', 'number-answers', 'fixed-order', 'inline']
+    optional_attribs = ['weight', 'number-answers', 'fixed-order', 'inline', 'enable-nota']
     pl.check_attribs(element, required_attribs, optional_attribs)
     name = pl.get_string_attrib(element, 'answers-name')
+    enable_nota = pl.get_boolean_attrib(element, 'enable-nota', False)
```

Comment:
Lets add a `NONE_OF_THE_ABOVE_DEFAULT = False` constant to the top of the file and reference it here

Reply:
updated 👍